src/jit/compiler.cpp

```diff
@@ -4424,6 +4424,18 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, CORJIT_F
 
 #ifdef DEBUG
     fgDebugCheckLinks(compStressCompile(STRESS_REMORPH_TREES, 50));
+
+    // Stash the current estimate of the function's size.
+    compSizeEstimate = 0;
+    compCycleEstimate = 0;
+    for (BasicBlock* block = fgFirstBB; block != nullptr; block = block->bbNext)
```

Comment:
Seems like these should be initialized earlier, along with "everything" in Compiler (compInit)